tests/functional/adapter/incremental_strategies/test_incremental_strategies.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+
+from dbt.tests.util import run_dbt, check_relations_equal
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
```

Comment:
do we need this `SeedConfigBase` import?

Reply:
I think doing the above and adding 

`@pytest.fixture(scope="class")
    def seeds(self):
        return {
            "expected_append": expected_append_csv,
            "expected_overwrite": expected_overwrite_csv,
            "expected_upsert": expected_upsert_csv,
            "expected_partial_upsert": expected_partial_upsert_csv
        }`
 to the base class might get us there not sure tests can see the seeds as it is now.

Reply:
I missed adding the seeds in. This might just work.